packages/neon/neon_files/lib/widgets/file_preview.dart

```diff
@@ -72,19 +70,17 @@ class FilePreview extends StatelessWidget {
   }
 }
 
-class FilePreviewImage extends NeonCachedImage {
+class FilePreviewImage extends NeonApiImage {
   factory FilePreviewImage({
-    required final Account account,
     required final FileDetails file,
     required final Size size,
   }) {
     final width = size.width.toInt();
     final height = size.height.toInt();
     final path = file.path.join('/');
-    final cacheKey = '${account.id}-preview-$path-$width-$height';
+    final cacheKey = 'preview-$path-$width-$height';
```

Comment:
This needs to stay as another account might have the same path, width and height but the image is actually different. Not critical since we don't leak any info as we are already on the client side, but still a problem.

Reply:
NeonApiImage does that for us :)

Reply:
Ah I overlooked that part.